RazorTooling.TestApps.sln

```diff
@@ -0,0 +1,40 @@
+﻿
```

Comment:
Separate sln is to prevent `./build verify` from breaking due to VS auto restore. Aka, VS is open working on RazorTooling, run functional tests, half way through VS will try to auto-restore which then breaks the currently running tests. By moving the test application out of the primary sln it deters breaks like this.